docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -192,9 +300,12 @@ fixture `My fixture`
 
 test('My test', async t => {
     const button = await elementWithId('my-button');
+	const div    = await Selector('.myDiv')(); 
 });
 ```
 
+Note that if selector initializer has several matching DOM nodes on the page the selector execution returns the first one from the set.
```

Comment:
Note that if a selector initializer has several matching DOM nodes on the page, the selector returns the first node from the matching set.